guides/common/modules/proc_upgrading-project-in-place-using-leapp.adoc

```diff
@@ -51,6 +73,44 @@ On {RHEL}, enable the `rhel-7-server-extras-rpms` repository:
 # {package-install-project} leapp leapp-repository
 ----
 
++
+. After the `leapp` and `leapp-repository` packages are installed, the machine does not need access to the {RHEL} 7 repositories anymore.
```

Comment:
I know I mentioned this in my notes, but I think it doesn't deserve an own step.

You can add it to the "setup RHEL7 repos" step as a comment, but I guess it's also fine to just ommit it.

Reply:
I can go ahead and omit this line.